setup.py

```diff
@@ -43,7 +44,7 @@
             'opencv-contrib-python>=4.4.0.42',
             'opencv-python>=4.4.0.42',
             'grpcio',
-            'diambra-engine',
+            'diambra-engine>2.1.x', # To make sure both protobuf <3.20 (TF 2.x current limitation) and >=3.20 are supported
```

Comment:
@discordianfish I am facing a problem with CICD here:
1) If I use `'diambra-engine>2.1.x'` CICD for tests works but release fails
2) If I use `'diambra-engine>2.1'` CICD for tests fails but release works

Locally, when I install the packages, both works... so really no idea.